docker/importer/importer.py

```diff
@@ -704,35 +704,44 @@ def _process_updates_rest(self, source_repo: osv.SourceRepository):
     """Process updates from REST API."""
```

Comment:
Could you please add a prose description of the intended logic to the docstring?

Reply:
Added in [2841c89](https://github.com/google/osv.dev/pull/2672/commits/2841c894b9213bdada5469c8ca1349aefbf547f5)